### PR TITLE
feat: Add application menus

### PR DESCRIPTION
**Pull request by erdkse**
This PR...

## Changes

* Add system menu for the application

## Fixes

*

## Checklist

- [x] tested locally
- [x] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: electron/menu.ts [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+};
+
+const helpMenu = (win: BrowserWindow, store: any): MenuItemConstructorOptions => {
+  return {
+    label: 'Help',
+    submenu: [
+      {
+        label: 'Documentation',
+        click: async () => {
+          await shell.openExternal('https://kubeshop.github.io/monokle/');
```
Comment (olensmar) [inline electron/menu.ts:216]: use the openDocumentation() function in shell.ts

File: electron/menu.ts [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+        },
+      },
+      {type: 'separator'},
+      {
+        label: 'Github',
+        click: async () => {
+          await shell.openExternal('https://github.com/kubeshop/monokle');
```
Comment (olensmar) [inline electron/menu.ts:223]: use the openGitHub() function in shell.ts

File: electron/menu.ts (hunk 1 above)
Comment (erdkse) [reply electron/menu.ts]: Done!

File: electron/menu.ts [hunk 3]
```diff
@@ -48,13 +48,16 @@ const fileMenu = (win: BrowserWindow, store: any): MenuItemConstructorOptions =>
       {
         label: 'Browse Folder',
         toolTip: BrowseFolderTooltip,
+        enabled: !(Boolean(mainState.previewResourceId) || Boolean(mainState.previewValuesFileId)),
```
Comment (olensmar) [inline electron/menu.ts:51]: maybe you can use the isInPreviewMode selector? (in selectors.ts)

**Comment by erdkse** [comment]
All changes are done!

**Review by olensmar — APPROVED** [review]

**Pull request merged**